README.md

```diff
@@ -17,33 +17,33 @@ Next, execute the database migrations/schema setup:
 	bundle exec rake db:setup
 
 
-### Initialize PhraseApp
+### Initialize Phrase
 
 Use the Rails generator to add a `phraseapp_in_context_editor.rb` initializer:
 
   bundle exec rails generate phraseapp_in_context_editor:install --access-token=<YOUR_ACCESS_TOKEN> --project-id=<YOUR_PROJECT_ID>
 
-If you don't have a [PhraseApp account](https://phraseapp.com/en/signup) yet, you can use the [demo account](https://phraseapp.com/demo) credentials to get started:
+If you don't have a [Phrase account](https://phrase.com/en/signup) yet, you can use the [demo account](https://phrase.com/demo) credentials to get started:
```

Comment:
"https://phrase.com/en/signup" still does not work. Not sure it is supposed to work

Reply:
this should be fixed by https://github.com/phrase/landingpage/pull/488